pipelines/broad/arrays/imputation_beagle/ImputationBeagle.wdl

```diff
@@ -152,23 +151,50 @@ workflow ImputationBeagle {
           for_dependency = FailQCNChunks.done # these shenanigans can be replaced with `after` in wdl 1.1
       }
 
-      call tasks.PhaseAndImputeBeagle {
+      # max amount of cpus you can ask for is 96 so at a max of 10k samples we can only ask for 9 cpu a sample
```

Comment:
would it make any sense to link to your doc here or in a README somewhere about how you got these numbers?